client/src/partials/accounts/create_account/create.js

```diff
@@ -135,6 +144,10 @@ angular.module('bhima.controllers')
       session.accountClass = $scope.checkClass($scope.newAccount.number);
```

Comment:
`$scope.checkClass()` is never called from the html view.  You don't need to attach it to the $scope.  Just make it a `function checkClass()`.